collection/zset/zset_test.go

```diff
@@ -493,6 +494,33 @@ func TestFloat64SetRemoveRangeByRank(t *testing.T) {
 	assert.Equal(t, N, z.Len()+len(actualNs))
 }
 
+func TestFloat64SetRemoveRangeByRankConcurrently(t *testing.T) {
+	const N = 10000
+	z := NewFloat64()
+	for i := 0; i < N; i++ {
+		z.Add(fastrand.Float64(), fmt.Sprint(i))
```

Comment:
```suggestion
		z.Add(float64(i), strconv.Itoa(i))
```
Using a static range will make debugging easier.